@czue build failure looks related
```
======================================================================
ERROR: corehq.apps.user_importer.tests.test_importer:TestUserBulkUpload.test_blank_is_active
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/apps/user_importer/tests/test_importer.py", line 233, in test_blank_is_active
    [],
  File "/vendor/lib/python3.6/site-packages/celery/local.py", line 193, in __call__
    return self._get_current_object()(*a, **kw)
  File "/vendor/lib/python3.6/site-packages/celery/app/task.py", line 379, in __call__
    return self.run(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/apps/user_importer/tasks.py", line 28, in import_users_and_groups
    update_progress=functools.partial(_update_progress, start=len(group_specs))
  File "/mnt/commcare-hq-ro/corehq/apps/user_importer/importer.py", line 368, in create_or_update_users_and_groups
    user.is_active = is_active
  File "jsonobject/base.pyx", line 327, in jsonobject.base.JsonObjectBase.__setattr__
  File "jsonobject/base_properties.pyx", line 79, in jsonobject.base_properties.JsonProperty.__set__
  File "jsonobject/base.pyx", line 299, in jsonobject.base.JsonObjectBase.__setitem__
  File "jsonobject/base.pyx", line 281, in jsonobject.base.JsonObjectBase.__unwrap
  File "jsonobject/base_properties.pyx", line 283, in jsonobject.base_properties.AssertTypeProperty.unwrap
  File "jsonobject/base_properties.pyx", line 269, in jsonobject.base_properties.AssertTypeProperty.assert_type
jsonobject.exceptions.BadValueError: '' not of type <class 'bool'>
----------------------------------------------------------------------
```

--- a/corehq/apps/user_importer/helpers.py
+++ b/corehq/apps/user_importer/helpers.py
@@ -5,4 +5,7 @@ def spec_value_to_boolean_or_none(user_spec_dict, key):
     value = user_spec_dict.get(key, None)
     if value and isinstance(value, str):
         return string_to_boolean(value)
-    return value
+    elif isinstance(value, bool):
+        return value
+    else:
+        return None

